images/fcos-bastion-image/root/usr/bin/prune_nodes

```diff
@@ -0,0 +1,202 @@
+#!/bin/bash
+
+function haproxy_cleanup() {
```

Comment:
change the function such that it doesn't use global vars and try to do the same with the others

Reply:
done